test/TritonGPU/automatic-warp-specialization.mlir

```diff
@@ -248,9 +248,9 @@ tt.func public @attention_forward(
 
     %P = arith.truncf %softmax : tensor<256x64xf32, #blocked> to tensor<256x64xf16, #blocked>
 
-    %P_tmem = ttng.tmem_alloc %P : (tensor<256x64xf16, #blocked>) -> !ttg.memdesc<256x64xf16, #tmem, #ttng.tensor_memory>
+    %P_smem = ttg.local_alloc %P : (tensor<256x64xf16, #blocked>) -> !ttg.memdesc<256x64xf16, #shared, #smem>
```

Comment:
could you also add fence check in lines 196-202?

Reply:
This is an e2e test `automatic-warp-specialization.mlir` running the whole pipeline, it's a bit awkward to do a local `CHECK` of a specific pass.